transport_layer_cc_test.go

```diff
@@ -0,0 +1,489 @@
+package rtcp
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestTransportLayerCC_RunLengthChunkUnmarshal(t *testing.T) {
+	for _, test := range []struct {
+		Name      string
+		Data      []byte
+		Want      RunLengthChunk
+		WantError error
+	}{
+		{
+			//3.1.3 example1: https://tools.ietf.org/html/draft-holmer-rmcat-transport-wide-cc-extensions-01#page-7
+			Name: "example1",
+			Data: []byte{0b00000000, 0b11011101},
```

Comment:
@adwpc Binary literals are introduced by Go 1.13.
CI is failing on Go 1.12.
https://travis-ci.org/pion/rtcp/jobs/649334284
```
# github.com/pion/rtcp
transport_layer_cc_test.go:18:18: missing ',' in composite literal
transport_layer_cc_test.go:38:3: missing ',' in composite literal
transport_layer_cc_test.go:40:3: missing ',' in composite literal
transport_layer_cc_test.go:41:38: missing ',' before newline in composite literal
transport_layer_cc_test.go:42:4: missing ',' before newline in composite literal
transport_layer_cc_test.go:43:3: expected operand, found 'if'
transport_layer_cc_test.go:44:70: missing ',' before newline in composite literal
transport_layer_cc_test.go:45:4: missing ',' before newline in composite literal
transport_layer_cc_test.go:46:3: missing ',' before newline in composite literal
transport_layer_cc_test.go:47:2: missing ',' before newline in composite literal
transport_layer_cc_test.go:49:6: expected '(', found TestTransportLayerCC_RunLengthChunkMarshal
FAIL	github.com/pion/rtcp [setup failed]
```